src/s390x/zipl.rs

```diff
@@ -29,22 +40,196 @@ pub fn chreipl<P: AsRef<Path>>(dev: P) -> Result<()> {
     Ok(())
 }
 
-/// Runs `zipl` based on Ignition and BLS configuration in `boot`.
-pub fn zipl<P: AsRef<Path>>(boot: P) -> Result<()> {
-    let boot = boot.as_ref();
+fn is_secure_execution() -> bool {
```

Comment:
The same libostree conversation around error-handling applies here. See e.g. [this code](https://github.com/coreos/coreos-installer/blob/8ab20393c388daecab08c05ed9c71a8bf40cf7e9/src/blockdev.rs#L1000) for how to gracefully handle `ENOENT`.